src/CloudNative.CloudEvents.AspNetCore/CloudNative.CloudEvents.AspNetCore.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageVersion>0.1</PackageVersion>
+    <Description>ASP.Net Core extensions for CloudNative.CloudEvents</Description>
+    <Copyright>Copyright Cloud Native Foundation</Copyright>
+    <RepositoryUrl>https://github.com/cloudevents/sdk-csharp</RepositoryUrl>
+    <PackageProjectUrl>https://cloudevents.io</PackageProjectUrl>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />

```

Comment:
Not sure if you're just copying what other projects in this repo do - but 2.2 is not a great version to target.

2.1 is an LTS release and is the last release that support .NET Framework (supported for a while)
2.2 is end of life in 3 months
3.X was just released and 3.1 (in preview) will be supported for 3 years

So either this should target 2.1 and include all of the currently supported releases, or target 3.0 for the new hotness.

Reply:
OR support both 2.1 and 3.0

Reply:
Changed target to latest 2.1 patch, didn't find an straight forward way to target multiple dependencies, you guys have any pointers for me? This works for 3.0 tho, so not sure if necessary.